packages/extension/src/Popup.js

```diff
@@ -32,8 +32,7 @@ function Popup() {
     chrome.tabs &&
       chrome.tabs.query(
         {
-          active: true,
-          currentWindow: false,
+          url: "http://localhost:3000/*"
```

Comment:
Need to look into this so we don't rely on hard coded url

Reply:
I changed this to passing around the host tabId and use that to send messages